packages/salesforcedx-vscode-core/src/metadataSupport/metadataDocumentationService.ts

```diff
@@ -0,0 +1,720 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+export type MetadataFieldInfo = {
+  name: string;
+  type: string;
+  description: string;
+  required?: boolean;
+};
+
+export type MetadataFieldDocumentation = {
+  name: string;
```

Comment:
don't duplicate 4 fields just to add 1

= MetadataFieldInfo & {validValues?: string[]}